index.js

```diff
@@ -192,6 +215,17 @@ client.on('message', async msg => {
 			} else {
 				msg.reply("isn't currently mentoring to begin with")
 			}
+		} else if (msg.content.toLowerCase().startsWith("!brb")) {
+			if (online) {
```

Comment:
This if is not closed and is causing execution issues

Reply:
Merge conflict stole my curly brace... should be fixed

Reply:
Merge conflict stole my curly brace... should be fixed